Mods/Alpakit/Source/Alpakit/Private/AlpakitModEntryList.cpp

```diff
@@ -72,6 +76,7 @@ void SAlpakitModEntryList::Construct(const FArguments& Args) {
                 SNew(SVerticalBox)
                     +SVerticalBox::Slot().AutoHeight().HAlign(HAlign_Center).Padding(0, 0, 0, 5)[
                         SNew(STextBlock)
+                        .ToolTipText(LOCTEXT("ReleaseTargets_Tooltip", "Select what target platforms this mod has custom C++ implementations for"))
```

Comment:
These aren't just for C++, these are the targets Alpakit will pack the mod for when in release mode